### PR TITLE
Removed all TODOs

### PR DESCRIPTION
**Pull request by LMiljak**
<layout

### CODE REVIEW

File: src/main/java/com/github/fishio/control/MainMenuController.java [hunk 1]
```diff
@@ -85,8 +85,6 @@ public void startMultiplayer() {
 	@FXML
 	public void loadLevel() {
 		AudioEngine.getInstance().playButtonSound();
-		
-		//TODO (not now)
 	}
```
Comment (Taeir) [inline src/main/java/com/github/fishio/control/MainMenuController.java]: Please remove the entire method, it is not used.

<organisation

**Comment by Taeir** [comment]
Looks good, most of those TODO's have already been resolved, or are no longer applicable.

<layout

**Comment by jessearens** [comment]
Can be merged if @Taeir 's suggestions are implemented/looked into.

<layout

**Comment by LMiljak** [comment]
Implemented @Taeir 's suggestion.

<interface

**Pull request merged**